### ISSUE TITLE
shapefile location

### ISSUE THREAD

**Issue by hdugan** [LimnoDataScience/plume_bloom_drivers#13]
Running targets, I get caught up on Last error: _paths_ missing files: 1_download/in/LakeSuperiorWatershed.shp
Assuming I need to have this shapefile locally?

**Comment by lindsayplatt** [comment]
Correct! It should be accessible in our Box folder (I just sent you an invite). See instructions here: https://github.com/LimnoDataScience/plume_bloom_drivers#lake-superior-spatial-data
